File: Better_Calculator.c
#include <stdio.h>
#include <stdlib.h>




int main()
{
    double x , y ; 
    char op ; 
    printf("Please enter the first number :") ; 
    scanf("%lf",&x) ;
    printf("Please enter the operation you want to use :") ; 
    scanf(" %c",&op) ;
    printf("Please enter the second number :") ; 
    scanf("%lf",&y) ;
    switch(op)
    {
        case '+' :
            printf("The sum of the two numbers = %lf",x+y) ;
            break ;
        case '-' :
            printf("The difference between the two numbers = %lf",x-y) ;
            break ;
        case '*' :
            printf("The product of the two numbers = %lf",x*y) ;
           break ;
        case '/' :
            printf("The first number divided by the second number = %lf",x/y) ;
            break ;
        default :
            printf("Invalid operation") ;
        break ;
    }
    return 0;
}

File: Squad-asl_AbdelrahmanMohamed_Python/Better_Calculator.py
from math import *
Number1 = input("Please enter the first number :")
Op = input("Please enter an operator :")
if Op == "!" :
    result = factorial(int(Number1) )
    print("The factorial of the number = " + str(result))
else :
    Number2 = input("Please enter the second number :")
    if Op == "+" :
        result = float(Number1) + float(Number2)
        print("The sum of the two numbers = " + str(result))
    elif Op == "-":
        result = float(Number1) - float(Number2)
        print("The difference between the two numbers = " + str(result))
    elif Op == "*":
        result = float(Number1) * float(Number2)
        print("The product of the two numbers = " + str(result))
    elif Op == "/":
        result = float(Number1) / float(Number2)
        print("The first number divided by the second number = " + str(result))
    else :
        print("Invalid operator")
